pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample

```diff
@@ -182,6 +182,11 @@ export WIFIPASS='your_pass'
 # export MATRIX_PASSWORD='put_your_matrix_password_here'
 # export MATRIX_ROOM='put_the_matrix_target_room_id_here'
 
+# Uncomment if setting up shell notifications. Use ":" for no action.
```

Comment:
Remove the bit about using ":" and just allow the variable to not be set, or left blank